src/sqlacodegen/generators.py

```diff
@@ -1100,7 +1100,9 @@ def render_models(self, models: list[Model]) -> str:
 
     def render_class(self, model: ModelClass) -> str:
         sections: list[str] = []
-
+        comments = self.render_table_comment(model)
+        if comments:
+            sections.append(comments)
         # Render class variables / special declarations
```

Comment:
Add a blank line after a control block ends.
```suggestion
            sections.append(comments)

        # Render class variables / special declarations
```